pkg/controller/v1alpha2/experiment/experiment_controller.go

```diff
@@ -36,12 +37,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/builder"
 
-	experimentsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
-	trialsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/trial/v1alpha2"
+	experimentv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
```

Comment:
I feel, Kubebuilder recommends plural form of CR.  Ideally, it should have been `apis/experiments`, `apis/trials` as per Kubebuilder notation. I was planning to take this up post 0.6 release

Reply:
@johnugeorge SGTM. I reverted the change. Maybe we could rename the package in #495

Reply:
Sounds good